### ISSUE TITLE
Add missing properties to Issues Struct

### ISSUE THREAD

**Issue by DavidMBusey** [xanzy/go-gitlab#918]
Making an issue so I don't forget to do this.

See: https://docs.gitlab.com/ce/api/issues.html#single-issue
`Users on GitLab Premium will additionally see the epic property:`

I'll test it out on my production and once I have something functioning, I'll commit it.

```
{
   "project_id" : 4,
   "description" : "Omnis vero earum sunt corporis dolor et placeat.",
   "epic_iid" : 5, //deprecated, use `iid` of the `epic` attribute
   "epic": {
     "id" : 42,
     "iid" : 5,
     "title": "My epic epic",
     "url" : "/groups/h5bp/-/epics/5",
     "group_id": 8
   },
   // ...
}
```

**Comment by DavidMBusey** [comment]
See: https://github.com/xanzy/go-gitlab/pull/917 MR.

**Comment by svanharmelen** [comment]
Fixes by #917
